Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,30 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+.Top{
+    background-image:url('../images/flowershop.jpg');
+    filter: brightness(80%);
+    display: flexbox;
+    width:1280px;
```

Comment:
I would not limit the size of your page with a hard coded width and height. 
Great use of the filter to educe the brightness